models/questions.sql

```diff
@@ -14,10 +14,16 @@ CREATE TABLE IF NOT EXISTS questions (
     topic_id BIGINT REFERENCES topics ON DELETE SET NULL ON UPDATE CASCADE,
     deleted_at TIMESTAMP WITH TIME ZONE,
     external_grading_enabled BOOLEAN DEFAULT FALSE,
-    external_grading_autograder text,
-    external_grading_environment text,
     external_grading_image text,
+    external_grading_files TEXT[] DEFAULT ARRAY[]::TEXT[],
+    external_grading_entrypoint text,
     UNIQUE (course_id, number)
 );
 
+ALTER TABLE questions DROP COLUMN IF EXISTS external_grading_autograder;
```

Comment:
See the [new migration docs](https://github.com/PrairieLearn/PrairieLearn/blob/master/doc/dev-guide.md#db-schema-migrations). I changed it so we now change the CREATE TABLE statement as you have done, but we now put the migration statments (ALTER TABLE) in a separate file in `migrations`. So you can just stick these four ALTER TABLE statements in a single file called something like `migrations/037__questions__external_grading__change.sql`.